src/helix-ui/elements/HXAccordionPanelElement.js

```diff
@@ -50,15 +51,20 @@ export class HXAccordionPanelElement extends HXElement {
     }
 
     static get $observedAttributes () {
-        return [ 'open' ];
+        return [ 'disabled', 'open' ];
```

Comment:
We don't need to explicitly observe `disabled`, because it's [already being observed](https://github.com/rackerlabs/helix-ui/blob/master/src/helix-ui/elements/HXElement.js#L139) by HXElement.

We just need to add logic in `$onAttributeChange()`.